.appveyor.yml

```diff
@@ -7,45 +7,127 @@ version: '{build}-{branch}'
 environment:
   matrix:
     - PYTHON: 'C:\\Python27'
-    
+
 cache:
   - '%PYTHON%\Lib\site-packages'
+  - '%PYTHON%\Scripts'
+  - '%APPVEYOR_BUILD_FOLDER%\_build\installers\'
 
 install:
   # - ps: Start-FileDownload "https://github.com/appveyor/ci/raw/master/scripts/enable-rdp.ps1" -Timeout 60000
   # - enable-rdp.ps1
-    
+
   # HTML Help Workshop 1.32
   # - cinst html-help-workshop
-  
+
   # Microsoft Visual C++ Redistributable 2008 (version min: 9.0.21022.8)
   # - cinst vcredist2008
 
   - cmd: if not exist _build\installers md _build\installers
 
-  - ps: Start-FileDownload "http://www.innosetup.com/download.php/is-unicode.exe" -Timeout 60000 -FileName "_build\installers\is-unicode.exe"
-  - cmd: _build\installers\is-unicode.exe /SP /VERYSILENT /SUPPRESSMSGBOXES /NORESTART /RESTARTAPPLICATIONS /NOICONS
+  # --------------Downloads and Installs--------------
+  - ps: |
+      $BuildFolder = $Env:APPVEYOR_BUILD_FOLDER + "\_build\installers\"
+      $SitePackages = $Env:PYTHON + "\Lib\site-packages\"
+
+      " "
+      "------------ Installing  Inno Setup ------------"
+      " "
+      $InnoInstaller = $BuildFolder + "is-unicode.exe"
+      If ((Test-Path $InnoInstaller) -eq $False) {
+        $InnoURL = "http://www.innosetup.com/download.php/is-unicode.exe"
```

Comment:
use the following link: http://files.jrsoftware.org/is/5/innosetup-5.5.9-unicode.exe